Clarify source of nextcloud entity information (#22393)
home-assistant/home-assistant.github.io: source/_integrations/nextcloud.markdown

---
title: Nextcloud

> description: Instructions on how to integrate Nextcloud monitor api data into Home Assistant. Sensors and BinarySensors will be created for most of the datapoints that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides. To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`

Reviewer: I don't think this belongs to the meta description part here.

Please adjust the documentation below instead.

Author: 🤦‍♂️Oh, right. Should be all set now.

ha_integration_type: integration
---

> The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) information into Home Assistant. Sensors and BinarySensors will be created for most of the datapoints that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides. To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`.

Reviewer: Small tweak suggestion for readability

```suggestion
The `nextcloud` integration pulls summary [Nextcloud](https://nextcloud.com/) information into Home Assistant.

The integration provides sensors and binary sensors for most of the data points that the built-in Nextcloud [serverinfo app](https://github.com/nextcloud/serverinfo) provides.

To see which datapoints your nextcloud instance exposes, browse to this url: `https://<your_nextcloud_url>/ocs/v2.php/apps/serverinfo/api/v1/info?format=json`.
```

Author: Looks good to me 👍
